activesupport/lib/active_support/deprecation/disallowed.rb

```diff
@@ -23,9 +23,10 @@ def disallowed_warnings
       end
 
       private
-        def deprecation_disallowed?(message)
+        def deprecation_disallowed?(message, deprecator)
```

Comment:
We don't need to pass the deprecator, this module is included in `ActiveSupport::Deprecation`, and `self` is a instance. We can just check `gem_name` directly.